### ISSUE TITLE
Weird question. When I open NvimTree and close it, the Outline window size changes and keeps getting wider

### ISSUE THREAD

**Issue by JavaHello** [simrat39/symbols-outline.nvim#139]
![outline](https://user-images.githubusercontent.com/24937965/174214480-bf264405-6d84-480e-be83-4cee1d2b0468.gif)

**Comment by simrat39** [comment]
Looks like a nvim-tree issues, I can repro it with any split (with winfixwidth), not just symbols-outline, so I think you should report it there.

**Comment by SamHep0803** [comment]
yep i'm getting the same issue, haven't looked much into the workings of the plugin, could take a quick peak and see if i can spot what the issue could be.

**Comment by xbladesub** [comment]
same
